converter/converter.py

```diff
@@ -7,21 +8,21 @@
 # Convert temperature
 
 
-def cel_to_fahren(temperature_in_celsium):
+def cel_to_fahren(temperature_in_celsium: float):
     """
     Take temperature in Celsius and convert to Fahrenheit
     Temp: 20 C
     20 * (9/5) + 32 = 68 F
     """
-    fahren = (temperature_in_celsium * (9 / 5)) + 32
+    fahren = round(((temperature_in_celsium * (9 / 5)) + 32), 1)
     return fahren
 
 
-def fahren_to_cel(temperature_in_fahren):
+def fahren_to_cel(temperature_in_fahren: float):
```

Comment:
@MariaMaximova, please add typing for the returned value

Reply:
Resolved in this [commit](https://github.com/MariaMaximova/convert_temp/pull/9/commits/ea07c0df2d274ab51fb7e4346d05465b52c56248)